packages/nft/scripts/deploy.mjs

```diff
@@ -44,10 +52,29 @@ rl.question(q, async (answer) => {
       console.log(`export ${NFT}`)
       const tbc721 = new TBC721(computer)
       const modSpec = await tbc721.deploy()
+
+      console.log("Deploying Offer contract...")
+      console.log(`export ${Offer}`)
+      const offerHelper = new OfferHelper(computer)
+      const offerModSpec = await offerHelper.deploy()
+
+      console.log("Deploying Sale contract...")
+      console.log(`export ${Sale}`)
+      const saleHelper = new SaleHelper(computer)
+      const saleModSpec = await saleHelper.deploy()
+
+      console.log("Deploying Payment contract...")
+      console.log(`export ${Payment}`)
+      const paymentHelper = new PaymentHelper(computer)
+      const paymentModSpec = await paymentHelper.deploy()
+
       console.log(`
       Deploy successful
       \x1b[2mYou can copy this module specifier\x1b[0m
-      const nftModSpec = \x1b[2m'${modSpec}'\x1b[0m
+      export const nftModSpec = \x1b[2m'${modSpec}'\x1b[0m
```

Comment:
We should consider using "fs.writeFile" here to write directly to the .env file (after prompting the user for their ok)

Reply:
The file might have some other code as well, which might get override because of this